src/main/java/seedu/address/model/module/UniqueModuleList.java

```diff
@@ -105,7 +105,7 @@ public Module find(ModuleCode code) {
                 return mods[i];
             }
         }
-        throw new ModuleNotFoundException();
+        return null;
```

Comment:
JavaDocs at line 98 can be removed accordingly

Reply:
Ok removed.